systems/controllers/bodyMotionControlmex.cpp

```diff
@@ -70,16 +70,13 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
   assert(mxGetM(prhs[narg])==6); assert(mxGetN(prhs[narg])==1);
   Map< Vector6d > body_vdot_des(mxGetPrSafe(prhs[narg++]));
 
-  pdata->r->doKinematics(q,false,qd);
+  pdata->r->doKinematicsNew(q, qd);

```

Comment:
I'm guessing that you kept that around (and the forwardKinNew) because the interface changed.  but i think we just need to update this.  not many people are using the c++ api yet, and it's better to make it clean, no?